.github/workflows/actions.yml

```diff
@@ -58,3 +58,21 @@ jobs:
         with:
           # 👇 Chromatic projectToken, refer to the manage page to obtain it.
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+
+  check-up-to-date:
+    name: Check if PR is up to date with the main branch
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+        #Check if the PR is up to date with the main branch
+      - name: Check if branch is ahead of main
+        run: |
+            if ! git merge-base --is-ancestor origin/main ${{ github.event.pull_request.head.sha }};
+            then echo "This branch is not up to date. Please rebase or merge with main.";
```

Comment:
You may change this statement to say, "Please merge with main."

This is important because we don't want any developer to use rebase, as in rebase the commit history is re-written, that is something that we want to prevent at all cost.

Reply:
Done!